libjsonutils/cmake/FindRapidJSON.cmake

```diff
@@ -1,46 +1,55 @@
 #FindRapidJSON.cmake
-# 
+#
 # Finds the rapidjson library
 #
+# from http://rapidjson.org/
+#
 # This will define the following variables
 #
-#    RapidJSON_FOUND
-#    RapidJSON_INCLUDE_DIRS
+#    RAPIDJSON_FOUND
+#    RAPIDJSON_VERSION
+#    RAPIDJSON_INCLUDE_DIRS
```

Comment:
Why was the case changed? These should probably stay `RapidJSON`.

Reply:
Claus-MBP:example_exec clausklein$ egrep -w '[A-Z]+_FOUND' /opt/local/share/cmake-3.16/Modules/Find*.cmake -l | wc
    113     113    6014
Claus-MBP:example_exec clausklein$ egrep -w '\w+_FOUND' /opt/local/share/cmake-3.16/Modules/Find*.cmake -l | wc
    144     144    7688
Claus-MBP:example_exec clausklein$

Reply:
Note: the FindBoost module is deprecated! new versions install cmake config files.

Reply:
But I revert to CamelCase